ios/Video/RCTVideo.swift

```diff
@@ -1195,7 +1195,8 @@ class RCTVideo: UIView, RCTVideoPlayerViewControllerDelegate, RCTPlayerObserverH
 
     // MARK: - React View Management
 
-    func insertReactSubview(view: UIView!, atIndex: Int) {
+    override func insertReactSubview(_ view: UIView!, at atIndex: Int) {
+        super.insertReactSubview(subview, at:atIndex)
```

Comment:
I assume that `super.insertReactSubView` is going to add the `subview` to the `RCTVideo`. Does this mean that if `_controls == true` that the `subview` is now going to be added to both the `RCTVideo` and the `contentOverlayView`?

If you plan to call `super.insertReactSubview` you would probably only want to call that when `_controls == false`?

Reply:
In our case the _controls prop is always true. The super needs to be called as the `removeReactSubview` won't be triggered otherwise. I'll dig deeper into what exactly is happening here, but visually it seems to be working good.